Address Colin nits (#24)
FiloSottile/draft-irtf-cfrg-ristretto255: draft-irtf-cfrg-ristretto255-decaf448.md

identity is a generator, but for interoperability a canonical generator
is selected, which can be internally represented by the Curve25519
basepoint, enabling reuse of existing precomputation for scalar

> multiplication. This is its encoding using the method described in (#encoding255):

Reviewer: ```suggestion
multiplication. This is its encoding as produced by the function
specified in (#encoding255):
```

encoding and then re-decoding it. However, the internal
representations themselves might not be identical.

> Implementations **MAY** also perform byte comparisons on encoded
> representations elements for an equivalent, although less efficient, result.

Reviewer: ```suggestion
Implementations **MAY** also perform byte comparisons on the encodings
of group elements (produced by (#encoding255)) for an equivalent, although
less efficient, result.
```

identity is a generator, but for interoperability a canonical generator
is selected, which can be internally represented by the edwards448
basepoint, enabling reuse of existing precomputation for scalar

> multiplication. This is its encoding using the method described in (#encoding448):

Reviewer: ```suggestion
multiplication. This is its encoding as produced by the function
specified in (#encoding448):
```

encoding and then re-decoding it. However, the internal
representations themselves might not be identical.

> Implementations **MAY** also perform byte comparisons on encoded
> representations elements for an equivalent, although less efficient, result.

Reviewer: ```suggestion
Implementations **MAY** also perform byte comparisons on the encodings
of group elements (produced by (#encoding448)) for an equivalent, although
less efficient, result.
```
